apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/WriteDisposition.cs

```diff
@@ -35,6 +35,12 @@ public enum WriteDisposition
         /// Attempting to write data to a non-empty table fails with an error.
         /// </summary>
         [ApiValue("WRITE_EMPTY")]
-        WriteIfEmpty
+        WriteIfEmpty,
+	/// <summary>
```

Comment:
Please could you turn these tabs into spaces?

Reply:
I trusted vim, and it betrayed me.  Fixed.